scripts/travis/linux-x86_64/ubuntu-20.04.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+set -euxo pipefail
+
+scons -Q clean
+
+for c in gcc-7 clang-7 gcc-8 clang-8 gcc-9 clang-9
+do
+    scons -Q \
+          --enable-werror \
+          --enable-pulseaudio-modules \
+          --enable-tests \
+          --enable-benchmarks \
+          --enable-examples \
+          --build-3rdparty=openfec,pulseaudio,google-benchmark \
```

Comment:
Ubuntu 20 has uptodate google benchmark and it's included into our dockerfile, so let's remove google-benchmark from --build-3rdparty.

Reply:
Hello @gavv thanks for your quick review, i will make that change.

Also i notice the build for clang-7 is not working the pipeline, it seems to be a linker problem when building **pulseaudio-5.0**.

`/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__pow_finite'
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__log10_finite'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__pow_finite'
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__log10_finite'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:6304: pacat] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: *** [Makefile:6322: pactl] Error 1
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__pow_finite'
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__log10_finite'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:6334: pasuspender] Error 1
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__pow_finite'
/usr/bin/ld: ./.libs/libpulse.so: undefined reference to `__log10_finite'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:6316: pacmd] Error 1
make[3]: Leaving directory '/opt/roc/3rdparty/x86_64-pc-linux-gnu/clang-7.0.1-release/build/pulseaudio-5.0/src/pulseaudio-5.0/src'
make[2]: *** [Makefile:4655: all] Error 2
make[2]: Leaving directory '/opt/roc/3rdparty/x86_64-pc-linux-gnu/clang-7.0.1-release/build/pulseaudio-5.0/src/pulseaudio-5.0/src'
make[1]: *** [Makefile:756: all-recursive] Error 1
make[1]: Leaving directory '/opt/roc/3rdparty/x86_64-pc-linux-gnu/clang-7.0.1-release/build/pulseaudio-5.0/src/pulseaudio-5.0'
make: *** [Makefile:591: all] Error 2
`

Reply:
Roc tries to detect pulseaudio version installed on the system and falls back to 5.0 if it can't. 5.0 is a very old version, so let's just try a more up-to-date one.

Is pulseaudio installed in the image? If not, let's try to install it and see if it fixes the build.

Reply:
So for now: 
- I will verify if `pulseaudio ` is installed in travis-ubuntu:20.04 
- If not i will modify the dockerfile in local to install it.
- If that solves the problem. I will make another PR to roc dockerfiles to update the image

Reply:
Yep.

Reply:
The build for ubuntu 20.04 now passed on travis, but the whole Linux job took over 50 mins and the job was terminated by travis:

```
The job exceeded the maximum time limit for jobs, and has been terminated.
```

So we need to split the jobs.